src/Controls/PlanarControls.js

```diff
@@ -18,12 +18,14 @@ const mouseButtons = {
 // starting camera position and orientation target
 const startPosition = new THREE.Vector3();
 const startQuaternion = new THREE.Quaternion();
+// camera initial zoom value if orthographic
+let cameraInitialZoom = 0;
 
 // point under the cursor
 const pointUnderCursor = new THREE.Vector3();
 
 // control state
-const STATE = {
+export const STATE = {
```

Comment:
why you export STATE?

Reply:
Exporting `STATE` allows unit tests to access it.